repo/ref.go

```diff
@@ -15,9 +15,15 @@ type Refstore interface {
 	// PutRef adds a reference to the store. References must be complete with
 	// Peername, Name, and Path specified
 	PutRef(ref DatasetRef) error
+	// GetRef "completes" a passed in partial reference, filling in missing fields
```

Comment:
Does "partial" here mean that the input is allowed to be missing Peername or Name or Path? Or does it mean that those are required, but that GetRef will fill in the Dataset field?